src/ansys/fluent/core/services/datamodel_tui.py

```diff
@@ -330,13 +330,13 @@ def __dir__(self) -> list[str]:
             for x in PyMenu(
                 self._service, self._version, self._mode, self._path
             ).get_child_names()
-            if x != "exit"
+            if x not in ["exit", "switch_to_meshing_mode"]
         ]
 
     def __getattribute__(self, name) -> Any:
-        if name == "exit" and not self._path:
+        if name in ["exit", "switch_to_meshing_mode"] and not self._path:
```

Comment:
Tui is better handled from server-side. Please refer PR by @mkundu1

Reply:
OK, I had already pinged him

Reply:
This change is fine for PyFluent perspective. We'll consolidate the server-side changes in 25.2.